String Searching/wu_manber.cpp

```diff
@@ -0,0 +1,96 @@
+#include <bits/stdc++.h>
```

Comment:
please don't use **bits/stdc++.h** because this is quite Linux specific and slows down the compiler process.
_(Specified in [README](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/README.md))_